src/main/java/com/gbtec/Main.java

```diff
@@ -1,22 +1,59 @@
 package com.gbtec;
 
-import static com.gbtec.service.GenerationCalculator.calculateNextGeneration;
-import static com.gbtec.service.GenerationReader.createInitialGeneration;
-
-import com.gbtec.model.Generation;
-import com.gbtec.presentation.Printer;
 
 public class Main {
 
-    public static void main(String[] args) {
-        Printer printer = new Printer();
-        Generation currentGeneration = createInitialGeneration();
+    public static void main(String[] args) throws InterruptedException{
+        int sizeOfGeneration = 50;
+        int numberOfNextGenerations = 100;
+        int wishPattern = 1;
+
+        // Select which kind of pattern you want
+        Generation generation = new Generation(sizeOfGeneration);
+        if (wishPattern == 0) {
+            GenerationRandomizer.randomizeGeneration(generation, sizeOfGeneration);
+            // introducing the GenerationPrinter
+            GenerationPrinter printer2 = new GenerationPrinter();
+            printer2.printGeneration(generation, sizeOfGeneration);
 
-        printer.printGeneration(currentGeneration);
-        while (true) {
-            currentGeneration = calculateNextGeneration(currentGeneration);
-            printer.printGeneration(currentGeneration);
+            for (int i = 0; i <= numberOfNextGenerations - 1; i++) {
```

Comment:
This code snippet is used on 3 places. Please move this code in a separate method